code/modules/projectiles/projectile/beams.dm

```diff
@@ -216,6 +216,34 @@
 	impact_light_range = 2.5
 	impact_light_color_override = COLOR_LIME
 
+/obj/projectile/beam/emitter/hitscan/cascade
+	muzzle_type = /obj/effect/projectile/muzzle/laser/emitter/cascade
+	tracer_type = /obj/effect/projectile/tracer/laser/emitter/cascade
+	impact_type = /obj/effect/projectile/impact/laser/emitter/cascade
+	light_color = COLOR_MOSTLY_PURE_RED
+	hitscan_light_color_override = 	COLOR_MOSTLY_PURE_RED
+	muzzle_flash_color_override = COLOR_MOSTLY_PURE_RED
+	impact_light_color_override = COLOR_MOSTLY_PURE_RED
+	damage = 300
+
+/obj/projectile/beam/emitter/hitscan/cascade/on_hit(atom/target, blocked, pierce_hit)
+	if(isliving(target))
+		var/mob/living/mob_target = target
+		mob_target.dust()
+		return
```

Comment:
check armour here and only dust them if they don't have full laser protection or a certain amount of laser protection,

otherwise, just damage them a whole lot

Reply:
I want to make the laser do PASSMOB, the issue I'm trying to prevent here is borgs blocking the emitter because they don't naturally collapse on the floor

Reply:
> I want to make the laser do PASSMOB, the issue I'm trying to prevent here is borgs blocking the emitter because they don't naturally collapse on the floor

Dust mobs after a certain amount of damage/if they're dead?